Makefile

```diff
@@ -54,18 +54,23 @@ deploy: bootstrap ## Deploy all infrastructure
 	&& cd ${MAKEFILE_DIR}/infrastructure \
 	&& terragrunt run-all apply --terragrunt-non-interactive
 
-deploy-core: bootstrap ## Deploy core infrastructure
```

Comment:
We don't need to make `deploy-core` depend on `build-transform-library`, but `deploy` does need to depend on `build-transform-library`